spec/src/main/asciidoc/architecture.asciidoc

```diff
@@ -32,35 +32,35 @@ Only this Specification will be generated and maintained to document the content
 
 In reverse chronological order, here's the specification for each MicroProfile release.
 
-[[microprofile4.0-M1]]
-=== MicroProfile 4.0 Milestone 1 (Q32020)
-// MicroProfile 4.0-M1 is the 14th platform release for the
-// https://projects.eclipse.org/projects/technology.microprofile[Eclipse MicroProfile project].
-MicroProfile 4.0-M1 is based on Jakarta EE 8, the first MicroProfile release to
+[[microprofile4.0-RC1]]
+=== MicroProfile 4.0 Release Candidate 1 (Q42020)
+// MicroProfile 4.0-RC1 is the 14th platform release for the
```

Comment:
We have to figure out an alternate way of keeping track of our "releases".  The Mx and RCx are not "releases" in the normal sense of the word.  Nobody should be using these Mx or RCx releases in production usage.  The term "release" seems to indicate that they could be.  

I think we are now past the point of counting our releases.  We should just indicate the "next", and then maybe qualify the release with the words "milestone" or "release candidate" to make it clear what the intent is.